### PR TITLE
update paging, editing and upgrdade guide topics

### PR DESCRIPTION
**Pull request by hanastasov**
Closes # 

### Checklist:
 
 - [ ] check topic's TOC/menu and paragraph headings
 - [ ] link to other topics using `../relative/path.md`
 - [ ] at the References section at the end of the topic add links to topics, samples, etc
 - [ ] reference API documentation instead of adding a section with API

<br />

 - [ ] use valid component names - [Data] Grid, `IgxSelectComponent`, `<igx-combo>`
 - [ ] use spell checker tool (VS Code, Grammarly, Microsoft Editor)
 - [ ] add inline `code blocks` for the names of classes / tags / properties
 - [ ] add language descriptor for the ```code blocks```
 - [ ] check broken links (use browser add-on)
 - [ ] check if sample is working and fully visible in the topic
 - [ ] check if sample is working and fully visible in the StackBlitz
 - [ ] check if code blocks match the code in StackBlitz demo

<br />

 - [ ] follow SEO guidelines to fill topic's metadata ([SEO guidelines](https://infragisticsinc297.sharepoint.com/:w:/g/Groups/marketing/EWz9InT4FDlErHCumxsKGY4Bd8H03yhRWxDFk47luRz-_Q?e=S5wWcx))

<br />

 - [ ] do not resolve requested changes (leave that to the reviewer)
 - [ ] add `pending-localization` label when the review of the PR is done
 - [ ] add a member from the localization team to translate it

### CODE REVIEW

File: en/components/grids_templates/paging.md [hunk 1]
```diff
@@ -55,97 +55,141 @@ The following example represents @@igComponent pagination and exposes the option
...
+    const start = page * perPage;
+    const end = (page + 1) * perPage;
+    const result = localData.slice(start, end);
+    return result;
+}
+```
 
-// Enables/disables paging
-this.@@igObjectRef.paging = false;
+or pipe the `data` input value through a custom Angular pipe - this way a developer need not worry about change detection in certain scenarios, Angular will handle it internally:
```
Comment (zdrawku) [inline en/components/grids_templates/paging.md:103]: Let's start with uppercase, or something is missing from the beginning of the sentence?

Comment (zdrawku) [inline en/components/grids_templates/paging.md:103]: a developer need not worry about 🤔  hm

File: en/components/grids_templates/remote-data-operations.md [hunk 2]
```diff
@@ -789,7 +709,7 @@ The last step will be to declare our template for the gird.
 After all the changes above, the following result will be achieved.
 
 @@if (igxName === 'IgxGrid') {
-<div class="sample-container loading" style="height:620px">
+<div class="sample-container loading" style="height:400px">
```
Comment (zdrawku) [inline en/components/grids_templates/remote-data-operations.md:712]: WIth 400px the sample is not fully visible:

image

**Review by zdrawku — COMMENTED** [review]
Also, please mark all relevant topics with 'updated' tag in the TOC

File: en/components/general/update-guide.md [hunk 3]
```diff
@@ -1,320 +1,327 @@
...
+
+
+Unfortunately not all changes can be automatically updated. Changes bellow are split into sections as they occur in the versions, so if any updates are required you should start from your current version and apply further updates from bottom to top.
+
+For example: if you are updating from version 6.2.4 to 7.1.0 you'd start from the "From 6.x .." section apply those changes and work your way up:
+
+## From 11.0.x to 11.1.x
+* IgxGrid, IgxTreeGrid, IgxHierarchicalGrid
+    * The `onPagingDone` output is removed from the grid. Other paging inputs/outputs like `page`, `perPage`, `totalPages`, `nextPage`, `previousPage`, `paginate` are deprecated and will also be removed in a future major release.
+    * If any of these are used in your application, you will need to migrate to using an external paginator for the grid and use the corresponding inputs/outputs exposed by the external paginator.
```
Comment (zdrawku) [inline en/components/general/update-guide.md:47]: Add a reference to the topic that shows that.

**Comment by kdinev** [comment]
@hanastasov Please update the theming snippets for the pager together with your changes #2573

File: en/components/grids_templates/paging.md (hunk 1 above)
Comment (hanastasov) [reply en/components/grids_templates/paging.md:111]: it is correct, i prefer it as it is shorter than "does not need to"

**Pull request merged**